lib/transaction.js

```diff
@@ -674,8 +674,15 @@ export default class Transaction {
   
     const inInputs = u =>
       inputs.findIndex(input => u.outpoint.equals(input.prevout)) > -1;
-    const sum = unspent
-      .filter(inInputs)
+    const inputUtxos = unspent.filter(inInputs);
+
+    if (Util.isNFT(color) || Util.isNST(color)) {
```

Comment:
need `isNST` here only because of [this](https://github.com/leapdao/leap-core/pull/81/files#diff-3deb3f32958bb937ae05c6f3e4abbdf5R165).